<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GnomeSettingsPage</class>
 <widget class="QWidget" name="GnomeSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gnome Settings</string>
  </property>
  <layout class="QGridLayout" name="contentGridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="horizontalSpacing">
    <number>20</number>
   </property>
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="menuLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnUnitySettings">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Unity Settings</string>
       </property>
       <property name="icon">
        <iconset resource="../../static.qrc">
         <normaloff>:/static/themes/common/img/ubuntu.png</normaloff>:/static/themes/common/img/ubuntu.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsTopButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWindowManager">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Window Manager</string>
       </property>
       <property name="icon">
        <iconset resource="../../static.qrc">
         <normaloff>:/static/themes/common/img/window.png</normaloff>:/static/themes/common/img/window.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsTopButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAppearance">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Appearance</string>
       </property>
       <property name="icon">
        <iconset resource="../../static.qrc">
         <normaloff>:/static/themes/common/img/appearance.png</normaloff>:/static/themes/common/img/appearance.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">settingsTopButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../static.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="settingsTopButtonGroup"/>
 </buttongroups>
</ui>
